packages/opentelemetry-test-utils/package.json

```diff
@@ -38,18 +38,18 @@
     "@opentelemetry/api": "^1.0.2"
   },
   "devDependencies": {
-    "@opentelemetry/api": "1.0.2",
+    "@opentelemetry/api": "1.1.0",
     "@types/node": "16.11.21",
     "gts": "3.1.0",
     "typescript": "4.3.5"
   },
   "dependencies": {
-    "@opentelemetry/core": "1.0.1",
-    "@opentelemetry/exporter-jaeger": "1.0.1",
-    "@opentelemetry/instrumentation": "0.27.0",
-    "@opentelemetry/sdk-trace-node": "1.0.1",
-    "@opentelemetry/resources": "1.0.1",
-    "@opentelemetry/sdk-trace-base": "1.0.1",
-    "@opentelemetry/semantic-conventions": "1.0.1"
+    "@opentelemetry/core": "^1.1.1",
+    "@opentelemetry/exporter-jaeger": "^1.1.1",
+    "@opentelemetry/instrumentation": "^0.27.0",
+    "@opentelemetry/sdk-trace-node": "^1.1.1",
+    "@opentelemetry/resources": "^1.1.1",
+    "@opentelemetry/sdk-trace-base": "^1.1.1",
+    "@opentelemetry/semantic-conventions": "^1.0.1"
```

Comment:
Why is here `^1.0.1` but the others are at `^1.1.1`?
At other places you moved to 1.1.x only for dep-dependencies but not for prod dependencies. So maybe it should be `^1.0.0`?

Reply:
I only updated core packages that rely on api, semantic conventions does not.
Since it was `1.0.1` before, I didn't want to to downgrade the possible versions and just opened it up for upgrades.